core/src/main/java/com/linecorp/armeria/server/docs/EnumInfo.java

```diff
@@ -39,6 +41,28 @@
     @Nullable
     private final String docString;
 
+    /**
+     * Creates an {@link EnumInfo} from the specified {@link Class}.
+     */
+    public static EnumInfo of(Class<?> enumClass) {
```

Comment:
- `Class<? extends Enum<?>>`
- Could you make this a constructor or make all other constructors static factory methods? (consistency)